supplementary/devices/index.rst

```diff
@@ -121,7 +121,7 @@ Devices
      - Navigation
      - Sensors
      - Other
-   * - :doc:`Samsung Galaxy S8+ <dream2lte>`
+   * - `Samsung Galaxy S8+ <dream2lte>`
```

Comment:
I think here is one place left where the `:doc:` gets removed, while we should keep it